src/testdir/test_hardcopy.vim

```diff
@@ -63,6 +63,35 @@ func Test_printmbfont()
   bwipe
 endfunc
 
+func Test_printmbcharset()
+  " FIXME: Unclear why this fails on Windows.
+  if !has('unix')
+    return
+  endif
```

Comment:
The test was hanging on Window CI (Appveyor).
I merely disabled the new tests on Windows
as I don't have a Windows machine to reproduce it.

I'd appreciate if someone with a windows host could check
why this test is failing.  File `test_hardcopy.vim` has several
tests disabled on Windows:
* Test_printmbcharset()
* Test_printexpr()
* Test_errors()
* Test_empty_buffer()